<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>43</red>
        <green>43</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>93</red>
        <green>93</green>
        <blue>93</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>77</red>
        <green>77</green>
        <blue>77</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>31</red>
        <green>31</green>
        <blue>31</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>41</red>
        <green>41</green>
        <blue>41</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>43</red>
        <green>43</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>43</red>
        <green>43</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>31</red>
        <green>31</green>
        <blue>31</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>43</red>
        <green>43</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>93</red>
        <green>93</green>
        <blue>93</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>77</red>
        <green>77</green>
        <blue>77</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>31</red>
        <green>31</green>
        <blue>31</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>41</red>
        <green>41</green>
        <blue>41</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>43</red>
        <green>43</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>43</red>
        <green>43</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>31</red>
        <green>31</green>
        <blue>31</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>31</red>
        <green>31</green>
        <blue>31</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>43</red>
        <green>43</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>93</red>
        <green>93</green>
        <blue>93</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>77</red>
        <green>77</green>
        <blue>77</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>31</red>
        <green>31</green>
        <blue>31</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>41</red>
        <green>41</green>
        <blue>41</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>31</red>
        <green>31</green>
        <blue>31</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>31</red>
        <green>31</green>
        <blue>31</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>43</red>
        <green>43</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>43</red>
        <green>43</green>
        <blue>43</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>62</red>
        <green>62</green>
        <blue>62</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Estacionamiento</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/then.png</normaloff>../../../Downloads/then.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color: rgb(43, 43, 43)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbl_infoC">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 170, 0)</string>
      </property>
      <property name="text">
       <string>Informacion de Cliente</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lbl_ingreso">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0)</string>
          </property>
          <property name="text">
           <string>Ingreso:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_ingr_date">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>00/00/00</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="lbl_egreso">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0)</string>
            </property>
            <property name="text">
             <string>Egreso:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_egre_date">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255)</string>
            </property>
            <property name="text">
             <string>00/00/00</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="lbl_tiempo_playa">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0)</string>
            </property>
            <property name="text">
             <string>Tiempo en la playa:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_tplaya">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255)</string>
            </property>
            <property name="text">
             <string>0 Dias 00:00:00 Horas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_patente_display">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0)</string>
            </property>
            <property name="text">
             <string>Patente:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_patente">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255)</string>
            </property>
            <property name="text">
             <string>AAA000</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="lbl_monto">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0)</string>
            </property>
            <property name="text">
             <string>Monto Total:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblmoney">
            <property name="font">
             <font>
              <pointsize>24</pointsize>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255)</string>
            </property>
            <property name="text">
             <string>$ 00.00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_infoP">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 0)</string>
        </property>
        <property name="text">
         <string>Informacion de la Playa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="barra">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_cant_autos">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Hay 0 Autos en la playa de 0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="lbltime">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>00/00/00 00:00:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_actual">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>Actualizar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_space">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_esc_ticket">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>Escanear Ticket</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="lbl_nticketD">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0)</string>
          </property>
          <property name="text">
           <string>N de ticket:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_nticket">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_patente_dysplay2">
          <property name="styleSheet">
           <string notr="true">color: rgb(0,0, 0)</string>
          </property>
          <property name="text">
           <string>Patente:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtpatente">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_gentick">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>Generar Ticket</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAjustes">
    <property name="title">
     <string>Ajustes</string>
    </property>
    <addaction name="cambiardatos"/>
    <addaction name="separator"/>
    <addaction name="actualizar"/>
    <addaction name="close"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAjustes"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="cambiardatos">
   <property name="text">
    <string>Cambiar Datos</string>
   </property>
  </action>
  <action name="actualizar">
   <property name="text">
    <string>Actualizar</string>
   </property>
  </action>
  <action name="sesion">
   <property name="text">
    <string>Cerrar Sesion</string>
   </property>
  </action>
  <action name="close">
   <property name="text">
    <string>Cerrar Sesion</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>txtpatente</tabstop>
  <tabstop>btn_gentick</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_gentick</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>generar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_esc_ticket</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>escanear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtpatente</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>generar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>679</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_actual</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>actualizardata()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>generar()</slot>
  <slot>escanear()</slot>
  <slot>actualizardata()</slot>
  <slot>changedata()</slot>
  <slot>reloj()</slot>
 </slots>
</ui>
